tests/constructors/test_hamiltonians.py

```diff
@@ -1,28 +1,28 @@
 from __future__ import annotations
 
+from typing import Callable
+
 import networkx as nx
 import pytest
 from torch import rand
 
-from qadence import (
-    Interaction,
-    N,
-    Register,
-    X,
-    Y,
-    Z,
-    hamiltonian_factory,
-)
```

Comment:
I guess this was an automatic action?

Reply:
Yes some re-formatting in the pre-commit hooks